marklogic-data-hub-central/ui/src/components/queries/managing/manage-query-modal/manage-query.test.tsx

```diff
@@ -34,4 +34,9 @@ describe('Query Modal Component', () => {
     expect(queryByTitle('Delete')).not.toBeInTheDocument();
   });
 
-});
\ No newline at end of file
+  test('Verify link column does not exist', () => {
+    const { queryByTitle } = render(<ManageQuery modalVisibility={true} isSavedQueryUser={true} canExportQuery={true} queries={query} setQueries={jest.fn()} toggleApply={jest.fn()} queryName={'Lee'} setQueryName={jest.fn()} />);
+    expect(queryByTitle('Link')).toBeNull();
```

Comment:
@saarimrahman if you can, pls try and refactor these tests. The props passed to ManageQuery can be defined after `query` in the describe block and then passed into the test using a spread operator. Then overrides for any particular prop can be included, based on the requirement of the test.
Since all the tests here are one line, it would be great if we can group them together. Looking at the existing tests, they can be broadly classified into tests with a user who has permission and the other without.